src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -174,6 +174,9 @@ export class CalciteAccordionItem {
   updateActiveItemOnChange(event: CustomEvent): void {
     this.requestedAccordionItem = event.detail
       .requestedAccordionItem as HTMLCalciteAccordionItemElement;
+    if (this.el.parentNode !== this.requestedAccordionItem.parentNode) {
```

Comment:
Is there any chance `el.parentNode` or `requestedAccordionItem` could be undefined? I wonder if we need `this.requestedAccordionItem?.parentNode`.

Reply:
Good catch.

Reply:
Actually, looking at the event emitting, it looks like the item would always be there.

Reply:
I looked at the emitter like you mentioned, and the requestedAccordion item is the el, which is the host, correct? That wouldn't be undefined.